pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -93,6 +94,12 @@ stdenv.mkDerivation rec {
 
     echo "=== Running MathInstaller ==="
     ./MathInstaller -auto -createdir=y -execdir=$out/bin -targetdir=$out/libexec/Mathematica -silent
+
+    # Fix library paths
```

Comment:
Yet this might just work for now:
https://abi-laboratory.pro/tracker/timeline/zlib/index.html

The ABI change in 1.2.10 for deflate_copyright is probably just that 1.2.10 is one character longer than 1.2.9:
```
const char deflate_copyright[] =
   " deflate 1.2.8 Copyright 1995-2013 Jean-loup Gailly and Mark Adler ";
```